clubs/views.py

```diff
@@ -100,20 +102,21 @@ def perform_create(self, serializer):
             status=Club.PENDING
         )
         send_mail(
-            subject=email.CREATE_MAIL_SUBJECT,
-            message=email.CREATE_MAIL_BODY % {
+            subject=email.create_club.plaintext.subject.render(Context()),
+            message=email.create_club.plaintext.body.render(Context({
```

Comment:
what does this context-override do? (can we document that in the file for future readers/maintainers?)